docs/user-guide.md

```diff
@@ -24,34 +25,55 @@ This guide provides detailed instructions on how to configure your Liberty proje
+<img src="/docs/screenshots/update%20jdk%20toast.png" width="40%"/> 
+<img src="/docs/screenshots/java_17_toast_alert.png" width="40%" />
 
-<img src="/docs/screenshots/java_17_toast_alert.png" width="50%" height="50%">
+To resolve this issue, you can define properties in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) file or set system environment variables to point Liberty Tools to your Java 17 or later. 
 
-To resolve this conflict, define `java.jdt.ls.java.home` and `xml.java.home` in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings) file to point Liberty Tools to your Java 17 or later runtime.
+For both Liberty Config Language Server and Eclipse Language Server for Jakarta EE, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
+1. The [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) included by [Language Support for Java(TM) by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java). 
+  - By default, Liberty Tools installs the latest version of the Language Support for Java(TM) by Red Hat plugin. The latest version contains an embedded JRE higher than Java 17 and therefore no additional configuration is required. However, using an older version of the Language Support for Java(TM) by Red Hat plugin or using the universal version without the embedded JRE causes an issue, use `java.jdt.ls.java.home` to specify your new JDK path.
+2. `JDK_HOME` or `JAVA_HOME` as system environment variables. (Note: if both `JDK_HOME` and `JAVA_HOME` are set, `JDK_HOME` will take precedence)
+
+For LemMinX, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
+1. `xml.java.home` in settings.json
+2. `JDK_HOME` or `JAVA_HOME` as system environment variables. (Note: if both `JDK_HOME` and `JAVA_HOME` are set, `JDK_HOME` takes precedence)
 
 ![settings.json example](/docs/screenshots/settings.json%20path%20example.png)
 
-Liberty Tools for Visual Studio Code checks for the Java runtime in the following order of precedence:
-1. The [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) included by [redhat.java plugin](https://marketplace.visualstudio.com/items?itemName=redhat.java)
-2. `java.jdt.ls.java.home` in settings.json
-3. `java.home` in settings.json. Note: This setting is [deprecated!](https://github.com/redhat-developer/vscode-java#supported-vs-code-settings), use 'java.jdt.ls.java.home' instead. 
-4. `JDK_HOME`
-5. `JAVA_HOME`
+### External settings
+
+The following settings, which are provided by external extensions, are honoured when you run dev mode commands.
```

Comment:
I think we should remove this line. `xml.java.home` and `java.jdt.ls.java.home` are not connected to dev mode.